packages/gatsby-source-npm-package-search/src/gatsby-node.js

```diff
@@ -0,0 +1,79 @@
+const algoliasearch = require(`algoliasearch`)
+const crypto = require(`crypto`)
+
+var client = algoliasearch(`OFCNCOG2CU`, `f54e21fa3a2a0160595bb058179bfb1e`)
+var index = client.initIndex(`npm-search`)
+
+const createContentDigest = obj =>
+  crypto
+    .createHash(`md5`)
+    .update(JSON.stringify(obj))
+    .digest(`hex`)
+
+exports.sourceNodes = async ({ boundActionCreators }, { keywords }) => {
+  const { createNode } = boundActionCreators
+
+  console.log(`Grabbing NPM packages...`)
+
+  let buildFilter = []
+
+  keywords.forEach(keyword => {
+    buildFilter.push(`keywords:${keyword}`)
+  })
+
+  const data = await index.search({
+    query: ``,
+    filters: `(${buildFilter.join(` OR `)})`,
+    hitsPerPage: 1000,
+  })
+
+  // Currently the gatsby-transformer-remark plugin
```

Comment:
@gillkyle make sure to add this as a TODO to fix this in gatsby-remark-images or wherever the error is. Though I think @calcsam might have fixed this recently?